front/app/modules/commercial/id_vienna_saml/services/appConfiguration.ts

```diff
@@ -1,6 +1,6 @@
-import 'services/appConfiguration';
+import 'api/app_configuration/types';
```

Comment:
Shall we rename this file so it matches our new structure?

Reply:
front/app/modules/commercial/id_vienna_saml/services/appConfiguration.ts => front/app/modules/commercial/id_vienna_saml/api/app_configuration/types.ts

Reply:
Yes, good point!